src/components/data/SavedColumnSettings.js

```diff
@@ -67,7 +79,7 @@ const useSavedColumnSettings = ({ workspaceId, snapshotName, entityMetadata, ent
     const { namespace, name } = workspaceId
     await Ajax(signal).Workspaces.workspace(namespace, name).shallowMergeNewAttributes({
       // Store settings as a string since Rawls expects lists of objects to be entity references
```

Comment:
nit: does this comment need updating now?

Reply:
I am somewhat curious about what this comment means now. I will check with Nick on Monday since he will be back then

Reply:
I think this comment was written at an earlier stage of this feature's development, when attribute values looked like `[table name, visible, table name, visible, ...]`.

It looks like it was referring to the behavior of `shallowMergeNewAttributes`, which converts attribute values to attribute operations and in doing so, assumes lists of objects are entity reference lists.
https://github.com/DataBiosphere/terra-ui/blob/c514947dd99a14735323ab007f36ee5d82f91da1/src/libs/ajax.js#L881-L884
https://github.com/DataBiosphere/terra-ui/blob/c514947dd99a14735323ab007f36ee5d82f91da1/src/libs/ajax.js#L622-L636

Since the attribute value now is an object, I don't think this comment is relevant anymore.
https://github.com/DataBiosphere/terra-ui/blob/c9d1b60d2040dac05e192d0ee083ef004696aa42/src/components/data/SavedColumnSettings.js#L30-L49

Reply:
Ah thank you for digging up the history on this one!